docs/upgrade/upgrade-security.asciidoc

```diff
@@ -13,13 +13,25 @@ NOTE: For detailed information about the 8.0 release, check out the <<release-no
 [[upgrade-reqs]]
 == Upgrade requirements
 
-*Track rules that are automatically disabled when upgrading*
+[float]
+[[upgrade-agents]]
+=== Upgrade {agent}s
+
+Upgrade your {stack} and {agent}s to 7.17 first (refer to {fleet-guide}/upgrade-elastic-agent.html[Upgrade Fleet-managed Elastic Agents]). Afterwards, you can {stack-ref}/upgrading-elastic-stack.html[upgrade the {stack}] to 8.0. Initially, {agent}s will be version 7.17; this is fine because {elastic-sec} 8.x supports the last minor release in 7.x (7.17) and any subsequent {elastic-endpoint} versions in 8.x. After the {stack} upgrade, you can decide whether to upgrade {agent}s to 8.0, which is recommended to ensure you get the latest features.
```

Comment:
```suggestion
Upgrade your {stack} and {agent}s to 7.17 first (refer to {fleet-guide}/upgrade-elastic-agent.html[Upgrade Fleet-managed Elastic Agents]). Afterwards, you can {stack-ref}/upgrading-elastic-stack.html[upgrade the {stack}] to 8.x. Initially, {agent}s will be version 7.17; this is fine because {elastic-sec} 8.x supports the last minor release in 7.x (7.17) and any subsequent {elastic-endpoint} versions in 8.x. After the {stack} upgrade, you can decide whether to upgrade {agent}s to 8.0, which is recommended to ensure you get the latest features.
```

Reply:
Mainly recommending this because the [linked upgrade docs](https://security-docs_1663.docs-preview.app.elstc.co/guide/en/elastic-stack/master/upgrading-elastic-stack.html) are pointing to a topic that contains the 8.2 version, which leads me to assume that the link will lead users to the latest version of 8.x Stack upgrade docs.